src/Engine/GraphicsHandler/GraphicsHandler.cpp

```diff
@@ -25,28 +25,50 @@ GraphicsHandler::GraphicsHandler(SDL_Renderer& renderer)
 }) {
 }
 
-void GraphicsHandler::draw(SDL_Texture* texture, SDL_Rect texture_rect, GraphicPriority priority) {
-    draw_queue_.at(priority).push_back(DrawRequest(texture, texture_rect));
+void GraphicsHandler::draw(SDL_Texture* texture,
+                           SDL_Rect texture_rect,
+                           GraphicPriority priority,
+                           bool is_flyweight,
```

Comment:
I still want to rename to "cleanup"

Reply:
I swear i did this twice already. Thought this was done but I'll do it again. I'm kinda confused if i just forgot to commit changes or something..